test-automation/api-tests/test.py

```diff
@@ -69,21 +72,100 @@ def load_keys() -> None:
         for f in key_content:
             if f.startswith("."):
                 continue
-            if os.path.isabs(args.keys_dir):
-                key_file_path = f"{args.keys_dir}/{key}/{f}"
-            else:
-                key_file_path = f"{os.getcwd()}/{args.keys_dir}/{key}/{f}"
+            key_file_path = f"{os.path.abspath(args.keys_dir)}/{key}/{f}"
             try:
                 response = CLI.single_call(f"keys import-ks {key_file_path} {new_account_name} "
                                            f"--passphrase={args.passphrase}").strip()
                 if f"Imported keystore given account alias of `{new_account_name}`" == response:
                     ACC_NAMES_ADDED.append(new_account_name)
                     break
-            except RuntimeError as e:
+            except RuntimeError:
                 pass  # It's okay if import fails, just try next file in key's dir.
     assert len(ACC_NAMES_ADDED) > 1, "Must load at least 2 keys and must match CLI's keystore format"
 
 
+def is_after_epoch(n):
+    url = 'http://localhost:9500/'
+    payload = "{\n    \"jsonrpc\": \"2.0\",\n    \"method\": \"hmy_latestHeader\",\n    \"params\": " \
+              "[  ],\n    \"id\": 1\n}"
+    headers = {
+        'Content-Type': 'application/json'
+    }
+    try:
+        response = requests.request('POST', url, headers=headers, data=payload, allow_redirects=False, timeout=3)
+        body = json.loads(response.content)
+        return body["result"]["epoch"] > n
```

Comment:
probably 'epoch' will be an int, but future could change and this code doesn't take into account possibility of epoch coming back as string. Probably fine for now though